### PR TITLE
LPD-35599 | master

### PR DESCRIPTION
**Pull request by marcosapmf**
Jira ticket: https://liferay.atlassian.net/browse/LPD-35599

### CODE REVIEW

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:sf

File: modules/test/playwright/helpers/json-web-services/JSONWebServicesOSBAsahApiHelper.ts [hunk 1]
```diff
@@ -166,6 +205,41 @@ export class JSONWebServicesOSBAsahApiHelper {
 		);
 	}
 
+	async createBlogssDaily(blogsDaily: BlogDaily[]): Promise<any> {
```
Comment (interaminense) [inline modules/test/playwright/helpers/json-web-services/JSONWebServicesOSBAsahApiHelper.ts:208]: Hey @marcosapmf, there is a typo here `createBlogssDaily`

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:forward

**Pull request closed without merging**